### PR TITLE
PR17978 Follow-up - fix command

### PR DESCRIPTION
**Pull request by bmcelvee**
Follow-up to https://github.com/openshift/openshift-docs/pull/17978.

### CODE REVIEW

**Comment by bmcelvee** [comment]
@xiuwang I updated the command here. I left the `..5000` portion because I remember `..` being relevant. Please review when you get a chance. 

Thanks again!

File: modules/installation-restricted-network-samples.adoc [hunk 1]
```diff
@@ -64,7 +64,7 @@ $ oc image mirror registry.redhat.io/rhscl/ruby-25-rhel7:latest ${MIRROR_ADDR}/r
 configuration object:
 +
 ----
-$ oc create configmap registry-config --from-file={MIRROR_ADDR}=$path/ca.crt -n openshift-config
+$ oc create configmap registry-config --from-file={MIRROR_ADDR}..5000=$path/ca.crt -n openshift-config
```
Comment (xiuwang) [inline modules/installation-restricted-network-samples.adoc]: In step 2 we have used `${MIRROR_ADDR}`, it's better to use `${MIRROR_ADDR_HOSTNAME}..5000` to distinguish them.
And, it's better to give examples about those variables

**Comment by bmcelvee** [comment]
Thank you again! I'm going to go ahead and get this fix merged and look into adding examples.

**Comment by xiuwang** [comment]
@bmcelvee LGTM!
Thank you for all efforts!

**Pull request merged**